src/System.Collections.NonGeneric/tests/Comparer/Comparer_DefaultInvariant.cs

```diff
@@ -7,6 +7,7 @@
 using System.Globalization;
 using Xunit;
 
+[Collection("DefaultThreadCurrentCulture")]
```

Comment:
> I think it'd make sense to disable parallelism for the library until it can be entirely rewritten (which it should be at some point).

If tests will be rewritten they should also will rely on `CultureInfo.DefaultThreadCurrentCulture` and also will require deterministic execution like `[Collection]` does.

>None of these tests were written to be run in parallel, and I worry this is just one of potentially many such concurrency issues.

I think we shouldn't worry about that until it appears, but if it appear it'd be helpful for rewriting.